src/main/java/com/guciowons/shoppingbasket/Basket/BasketController.java

```diff
@@ -13,28 +13,38 @@ public BasketController(BasketService basketService) {
         this.basketService = basketService;
     }
 
-    @PutMapping(value = "/{id}/{quantity}")
-    public ResponseEntity<String> addProductToBasket(@PathVariable int id, @PathVariable int quantity){
+    @PostMapping
+    public ResponseEntity<String> createBasket(@RequestBody Basket basket){
```

Comment:
Tutaj raczej powinniśmy tworzyć koszyk bez podawania obiektu `Basket` -- to wymusza, że musisz podać jego `id`, które user ma sobie wymyślić? Co jak stworzysz koszyk z `id = 1`, dodasz do niego produkty, a potem stworzysz znowu koszyk z `id = 1`? Stracisz cały stan koszyka bo się nadpisze.

Id koszyka powinno się generować podczas tworzenia i być unikalne.